organization/documents.md

```diff
@@ -11,7 +11,13 @@ a full list of all of the papers connected to the [Community White Paper]({{ sit
 {:.table .table-hover .table-condensed .table-striped}
 | Title           | Purpose     | Download    | Source Link |
 | ------------- | ------------- | ------------ | ----------- |
-| HL-LHC Computing Review: Common Tools and Community Software | Submitted to the May 2020 LHCC HL-LHC review and (with minor amendments) to the US Snowmass Process | <https://zenodo.org/record/4009114> | [LaTeX](https://github.com/HSF/documents/tree/master/LHCC/2020/2020-01) |
-| Monte Carlo generators challenges and strategy towards HL-LHC | Submitted to the May 2020 LHCC HL-LHC review and to the US Snowmass Process | <https://arxiv.org/abs/2004.13687> | |
-| The Importance of Software and Computing to Particle Physics | Submitted to the European Strategy Update | <https://zenodo.org/record/2413005> | [LaTeX](https://github.com/HSF/documents/tree/master/HSF-DOC/2018-01) |
-| A Roadmap for HEP Software and Computing R&D for the 2020s | [Community White Paper]({{ site.baseurl }}/organization/cwp.html) document, also published in CSBS | <https://doi.org/10.1007/s41781-018-0018-8>, <https://arxiv.org/abs/1712.06982> | [LaTeX](https://github.com/HSF/documents/tree/master/CWP/papers/HSF-CWP-2017-01_roadmap/latex) |
+| The Critical Importance of Software for HEP | European Particle Physics Strategy Update 2025 | [Zenodo](https://doi.org/10.5281/zenodo.15097159) | [LaTeX](https://github.com/HSF/EPPSU-2025-Paper) |
+| PyHEP.dev 2024 Workshop Summary Report | PyHEP.dev 2024 Workshop | [arXiv](https://arxiv.org/abs/2410.02112) | |
+| Analysis Facilities White Paper | Community Analysis Facilities discussions | [arXiv](https://arxiv.org/abs/2404.02100) | HSF-TN-2024-01 |
```

Comment:
While adding this info I noticed that HSF-TN-2024-01 never made it to https://github.com/HSF/documents/tree/master/HSF-TN. We should resolve this as an issue and ping the editors.

Reply:
I resolved as https://github.com/HSF/hsf.github.io/issues/1723.